Firmware/menu.cpp

```diff
@@ -197,7 +179,7 @@ static void menu_draw_toggle_puts_P(const char* str, const char* toggle, const u
     sprintf_P(lineStr, PSTR("%c%-18.18S"), (settings & 0x01)?'>':' ', str);
     sprintf_P(lineStr + LCD_WIDTH - ((settings & 0x02)?strlen_P(toggle):strlen(toggle)) - 3, (settings & 0x02)?PSTR("[%S]%c"):PSTR("[%s]%c"), toggle, eol);
     if (!(settings & 0x04)) lcd_set_cursor(0, menu_row);
-    fputs(lineStr, lcdout);
+    lcd_print_pad(lineStr, LCD_WIDTH);
```

Comment:
Not sure how useful this is here. lineStr is still padded by the sprintf_P a bit higher. It might be worth rewriting this entire function at some point so it becomes more readable and optimal

Reply:
I'll re-check this one. I probably only added this because I saw a change in the memory footprint.

Reply:
Took a shot at some refactoring in [9f70de8](https://github.com/prusa3d/Prusa-Firmware/pull/3873/commits/9f70de8ba09b90f81e5745ec702d309808f20ee3) and saved 106 bytes. We can do a lot more refactoring but it will affect a lot more area in the code which I don't think we should attempt in this PR.